### PR TITLE
Support the newly added par_balanced_copied field.

### LINKED ISSUE #31: Support new field 'par_balanced_copied_bytes' of EVENT_GC_STATS_GHC
This field was added in [this commit](https://github.com/ghc/ghc/commit/7c9e356de1114ab3e31f2d6d03e83672076dd533).

I would like to add support for this to ghc-events, if you could provide some guidance on how you would like this done.

The options I see are:
* add a `Maybe Int` to the `GCStatsGC` constructor
* Add a GCStatsGC data type, with two constructors (perhaps GCStatsGC_V1, GCStatsGC_V2?) where the second has an additional field. This is my preference.

I would also like to add a function 
```haskell
--| Computes the cumulative parallel work balance of the argument garbage collections.   
workBalance :: [GCStatsGC] -> Double
```

### PR DESCRIPTION
**Pull request by sighingnow**
Add a fixed size parser that can handle the newer version of `GCStatsGHC` Event.

Fixes #31 and #41.

### CODE REVIEW

**Comment by maoe** [comment]
Could you add a small eventlog which contains the new field?

**Comment by sighingnow** [comment]
Done.

The `hello-ghc-x.x.x.eventlog` demonstrates the difference of `GCStatsGHC` generated by different version of GHC. The existing `stop.hT.eventlog` also have 58-bytes `GCStatsGHC` events and this PR have updated the `stop.hT.eventlog.reference` as well.

File: src/GHC/RTS/Events/Binary.hs [hunk 1]
```diff
@@ -154,6 +154,22 @@ standardParsers = [
       parNThreads  <- get :: Get Word32
       parMaxCopied <- get :: Get Word64
       parTotCopied <- get :: Get Word64
+      parBalancedCopied <- return Nothing
+      return GCStatsGHC{ gen = fromIntegral gen
+                       , parNThreads = fromIntegral parNThreads
+                       , ..}
```
[the remark this answers is not in the document]
Comment (sighingnow) [reply src/GHC/RTS/Events/Binary.hs]: Done. Thanks for the suggestion.

**Comment by maoe** [comment]
LGTM, thanks!

**Comment by maoe** [comment]
Just uploaded https://hackage.haskell.org/package/ghc-events-0.9.0.

**Comment by sighingnow** [comment]
@maoe Thanks!

**Pull request merged**